internal/tools/main.go

```diff
@@ -0,0 +1,28 @@
+// Copyright observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main exists to provide imports for tools used in development
```

Comment:
```suggestion
//go:build tools

// Package main exists to provide imports for tools used in development
```



This is what OTel does, following https://go.dev/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module. I think with this pattern we don't have to change the `Makefile` target

(also rename file to `tools.go`)

Reply:
(also changes package to `tools`)